service/src/main/java/greencity/service/NotificationServiceImpl.java

```diff
@@ -160,7 +161,8 @@ public void notifyHalfPaidPackage(Order order) {
         Set<NotificationParameter> parameters = new HashSet<>();
 
         Long paidAmount = order.getPayment().stream()
-            .filter(x -> !x.getPaymentStatus().equals(PaymentStatus.PAYMENT_REFUNDED))
+            .filter(x -> !x.getPaymentStatus().equals(PaymentStatus.PAYMENT_REFUNDED)
+                && !x.getPaymentStatus().equals(PaymentStatus.UNPAID))
```

Comment:
are you sure about &&? What case when we're got 2 payment statuses ??

Reply:
Find payment when PaymentStatus isn't PAYMENT_REFUNDED and isn't UNPAID.